packages/cli/package.json

```
@@ -40,7 +40,7 @@
     ]
   },
   "engines": {
-    "node": ">= 12"
+    "node": "14.x"
```

```suggestion
    "node": ">= 14"
```

We don't want to prevent users from using Node.js 16